src/utils/childProcess.js

```diff
@@ -31,9 +27,7 @@ export function runProcess(cmd) {
  */
 export function runCommand(cmd) {
     return new Promise((resolve, reject) => {
-        const commands = cmd.split(SPACE);
-        const [app, ...args] = commands;
-        const childProcess = spawn(app, args, { stdio: 'ignore' });
+        let childProcess = exec(cmd, { shell: true });
```

Comment:
According to [spec](https://nodejs.org/api/child_process.html#child_process_child_process_exec_command_options_callback) , `shell` flag in `exec` only takes a string, not a boolean like in `spawn`. There are also plenty of warning about using `shell` switch with uninitialized user input. Perhaps add some sanitization to strip out `>`,`&` and other potentially dangerous metacharacters?

Reply:
I ran tests w/o `shell: true` and they all passed. Does this mean test is not sufficient or that this flag is not necessary?

Reply:
Thanks a lot for this constructive feedback.
* About `shell` flag, good catch! I guess I kept that from one of dev iteration still using [`childProcess.spawn`](https://nodejs.org/api/child_process.html#child_process_child_process_spawn_command_args_options). I will simply remove it.
* Removing this, do you still recommend `cmd` sanitisation?